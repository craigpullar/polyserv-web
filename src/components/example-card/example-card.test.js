import ExampleCard from "./component";

describe("ExampleCard Component", () => {
  const componentProps = {
    classes: {
      card: {},
      title: {}
    }
  };

  const componentJSX = <ExampleCard {...componentProps} />;

  describe("Snapshot", () => {
    it("should match existing snapshot", () => {
      const componentSnapshot = global.renderer.create(componentJSX).toJSON();
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
