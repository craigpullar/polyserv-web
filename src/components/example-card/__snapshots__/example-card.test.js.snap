// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExampleCard Component Snapshot should match existing snapshot 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation1-5 MuiPaper-rounded-3 MuiCard-root-1"
>
  <h1
    className={Object {}}
  >
    EXAMPLE COMPONENT
  </h1>
</div>
`;
